src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import * as S from "./styles";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+    const [name, setName] = useState<string | null>(null);
+    const [subject, setSubject] = useState<string | null>(null);
+    const [content, setContent] = useState<string | null>(null);
+
+    const handleSubmit = (event: React.MouseEvent) => {
+      event.preventDefault();
```

Comment:
handleSubmit gets the current value of the state and adds it to the current webhooks.  event.preventDefault is called when submitting the form to prevent a browser reload/refresh